### DISCUSSION TITLE
alternative for aawirelles

### DISCUSSION THREAD

**Discussion by makbyr66** [AndreyPavlenko/Fermata#424] [General]
do you know if it could work on the Chinese EKIY A2 instead of aawirelless?

**Comment by lopropio** [comment]
Buy a better device, i used with Motorola MA1, android smartphone Motorola G100 and a VW Discover Media. Findings:
* Faster connect after radio ignition
*  Video playback without stuttering video streaming
*  Synchronous picture and sound playback
